### PR TITLE
Set up pythonparser

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

**Comment by nbirillo** [comment]
Now we can set up python parser automatically. But I found a problem, that TreeContext can not be created, But I have not found why. It usually happens with several lines files. I think we should check it with GumTree directly with different parsers, but the parsers itself work correctly...

**Comment by nbirillo** [comment]
Fixed it in [PR#7](https://github.com/hyperskill/code-submissions-clustering/pull/7), but changed the gumtree version...

File: code-submissions-clustering-core/src/main/resources/org/jetbrains/research/code/submissions/clustering/load/context/builder/gumtree/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+It is a folder to download and store a python parsder for GumTree via `GumTreeParserUtil`.
```
[the remark this answers is not in the document]
Comment (nbirillo) [reply code-submissions-clustering-core/src/main/resources/org/jetbrains/research/code/submissions/clustering/load/context/builder/gumtree/README.md:1]: Hm, but then we need to download the parser to the tmp folder, is it ok? If yes, I will do it @eartser

[the remark this answers is not in the document]
Comment (eartser) [reply code-submissions-clustering-core/src/main/resources/org/jetbrains/research/code/submissions/clustering/load/context/builder/gumtree/README.md:1]: Yes, this should work fine to my view

**Pull request merged**